openy.profile

```diff
@@ -229,6 +250,58 @@ function openy_install_features(array &$install_state) {
   return ['operations' => $module_operations];
 }
 
+/**
+ * Create batch for installing search.
+ *
+ * @param array $install_state
+ *   Installation parameters.
+ *
+ * @return array
+ *   Batch.
+ */
+function openy_install_search(array &$install_state) {
+  \Drupal::state()->set('openy_hide_solr_config', '1');
```

Comment:
I think we can rename variable to `openy_show_solr_config` and set it only once if solr is selected. So this line with a default value can be removed in this case.